corehq/apps/hqwebapp/tests/test_requirejs.py

```diff
@@ -21,29 +21,41 @@ def _run_jobs(cls, files, processor):
             jobs.append(gevent.spawn(processor, filename))
         gevent.joinall(jobs)
 
+    @classmethod
+    def _hqDefine_line(cls, filename):
+        pattern = re.compile(r'^\s*hqDefine')
+        with open(filename, 'r') as f:
+            for line in f:
+                if re.match(pattern, line):
+                    return line
+        return None
+
+
     @classmethod
     def setUpClass(cls):
         super(TestRequireJS, cls).setUpClass()
         prefix = os.path.join(os.getcwd(), 'corehq')
 
         proc = subprocess.Popen(["find", prefix, "-name", "*.js"], stdout=subprocess.PIPE)
         (out, err) = proc.communicate()
+        design_pattern = re.compile(r'/_design/')
+        couchapps_pattern = re.compile(r'couchapps')
+        vellum_pattern = re.compile(r'/vellum/')
         cls.js_files = [f for f in [b.decode('utf-8') for b in out.split(b"\n")] if f
-                    and not re.search(r'/_design/', f)
-                    and not re.search(r'couchapps', f)
-                    and not re.search(r'/vellum/', f)]
+                    and not design_pattern.search(f)
+                    and not couchapps_pattern.search(f)
+                    and not vellum_pattern.search(f)]
```

Comment:
nit: I think all of these could be changed to the form `literal not in f`, which should be more efficient than using regular expressions. For example, this one would be
```suggestion
                    and '/vellum/' not in f]
```

Reply:
616b974dab0f08d02124642744df303cacff8263